content/python/concepts/statsmodels/statsmodels.md

```diff
@@ -0,0 +1,146 @@
+---
+Title: 'Statsmodels'
+Description: 'The statsmodels library is used to estimate statistical models and perform statistical tests in Python.' 
+Subjects:
+  - 'Data Science'
+  - 'Statistics'
+Tags:
+  - 'Python'
+  - 'Statistics'
+  - 'Statsmodels'
```

Comment:
`Statsmodels` is not a valig tag. Please refer to the list of tags in [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

```suggestion
  - 'Data'
  - 'Models'
  - 'Datasets'
  - 'Machine Learning'
```